projects/demo/src/modules/components/tag/examples/4/index.ts

```diff
@@ -0,0 +1,22 @@
+import {Component} from '@angular/core';
+import {changeDetection} from '@demo/emulate/change-detection';
+import {encapsulation} from '@demo/emulate/encapsulation';
+
+@Component({
+    selector: 'tui-tag-example-4',
+    templateUrl: './index.html',
+    styleUrls: ['./index.less'],
+    changeDetection,
+    encapsulation,
+})
+export class TuiTagExample4 {
+    readonly tags = ['Taiga UI', 'is an open-source library', 'for awesome people'];
```

Comment:
```ts
tags: readonly string[] = ['Taiga UI', 'is an open-source library', 'for awesome people']
```

Reply:
The tags can be edited 🤔

Reply:
The objects/arrays must be immutable, for example you can create new array with new item (redux pattern).